apps/daimo-mobile/src/view/shared/SearchHeader.tsx

```diff
@@ -99,12 +110,13 @@ export function SearchHeader({
         </ButtonCircle>
       </Animated.View>
       <AnimatedSearchInput
+        ref={ref}
         icon="search"
         placeholder="Search for user..."
         value={prefix || ""}
         onChange={setPrefix}
-        onFocus={() => setPrefix("")}
-        onBlur={() => setPrefix(undefined)}
+        onFocus={() => setPrefix(prefix || "")}
```

Comment:
might not need to clear on focus anymore (now that we're clearing prefix when the back button is pressed)